packages/api/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.20.7]() (2019-10-30)
```

Comment:
I guess this should be `# [0.20.7]` not `## [0.20.7]`? I mean, the one below is like `# [0.20.6]`

Reply:
I made changes for 0.20.8 so I added it for 0.20.8.. 
I think 0.20.7 is missing.